src/common/types/objects/ledger_entries.ts

```diff
@@ -0,0 +1,56 @@
+import {SignerEntry} from './index'
+
+export type PayChannelLedgerEntry = {
```

Comment:
Why use `type` here instead of `interface`?

Reply:
@sublimator is right, that's a left over from Flow. Changing to interface...